packages/core/src/parsers/builder.ts

```diff
@@ -194,6 +195,37 @@ export const builderElementToJsxLiteNode = (
     properties.builderTag = block.tagName;
   }
 
+  const sizeKeys = Object.keys(block.responsiveStyles || {});
```

Comment:
```suggestion
  const blockSizes: Size[] = Object.keys(block.responsiveStyles || {}).filter(k => sizeNames.includes(k));
```
The idea would be do all the validation (excluding what's not a `size`) and the type casting (`string as Size`) in one line/step, then the rest of the code should be type-safe-_ish_. The thing is that later in the code we do `sizes[size as Size].max` but if `size` is not really a `Size`, this will crash.

Also renaming the `const` reads better (the sizes associated to this block).